tests/testsuite/serviceaccount.go

```diff
@@ -122,40 +131,10 @@ func createServiceAccount(saName string, clusterRole string) {
 	}
 }
 
-func cleanupServiceAccount(saName string) {
+func createSubresourceRole(saName string) {
```

Comment:
As is, `saName` parameter can be dropped!
Otherwise, I would suggest to keep it, adjusting `Role.Name`, `Role.Labels` and all the `RoleBindings` fields in the body function, to use it instead of the hardcoded value.
Up to you! You can just delete the parameter. I would not block the PR for this. Thanks!

Reply:
That was definitely an oversight on my part, thanks! Will fix it.